packages/experimental/src/global/js/utils/unwrap-if-fragment.js

```diff
@@ -1,14 +1,31 @@
 import React from 'react';
 
+/**
+ * unwrapIfFragment - walks the passed children unwrapping fragments
+ * to create a flat array of all children
+ */
 const unwrapIfFragment = (children) => {
   const newChildArray = [];
-  const realChildren =
-    children && children.type === React.Fragment
-      ? children.props.children
-      : children;
-  for (let index in realChildren) {
-    newChildArray.push(realChildren[index]);
+
+  const isFragment = (item) => item && item.type === React.Fragment;
+
+  const addChildren = (_children) => {
```

Comment:
I see what you're doing, with "_children" (although hiding the outer variable isn't actually a problem as we don't need it -- but it could get confusing!). But "_children" doesn't follow the variable name conventions we usually apply. Maybe use something like "items"?